cosmoz-tree-behavior.html

```diff
@@ -0,0 +1,62 @@
+<link rel="import" href="../polymer/polymer.html">
```

Comment:
Even though it's a nice way to wrap the tree stuff from a generic object, rather expect a Cosmoz.Tree type